dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
                 )
                 .getRequestItemAuthor(context, item);
 
-        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
+	// probably wrong -- Boolean.getBoolean() returns true if and only if
+	// the system property named by the argument exists and is equal to the
+	// string "true".
+        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
```

Comment:
done.

Thanks for all your help.  Clearly, I need to fix more bugs and become more
adept at the process. Hopefully it will be a little smoother next time!
Bill

On Fri, Aug 19, 2016 at 3:06 PM, Tim Donohue <notifications@github.com>
wrote:

> In dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/
> SendItemRequestAction.java
> <https://github.com/DSpace/DSpace/pull/1497#discussion_r75543911>:
>
> > @@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
> >                  )
> >                  .getRequestItemAuthor(context, item);
> >
> > -        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
> > +	// probably wrong -- Boolean.getBoolean() returns true if and only if
> > +	// the system property named by the argument exists and is equal to the
> > +	// string "true".
> > +        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
>
> Hi @wilee53 <https://github.com/wilee53> : You actually still can update
> this PR (based on this feedback). All you need to do is:
>
>    1. Checkout your "DS-3294-new" branch (git checkout DS-3294-new)
>    2. Make the changes
>    3. Commit them (git commit -a)
>    4. Finally do a git push origin DS-3294-new. This last statement will
>    push your changes up to your GitHub and *automatically updates this PR*
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DSpace/DSpace/pull/1497/files/679c971ec33126ebcbe6871835a7ac9ce408c931#r75543911>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJqhOdoGVj8ASi3NoIevOqg8_eJulyVFks5qhgzYgaJpZM4JouHX>
> .
>



-- 
Human wheels spin round and round
While the clock keeps the pace... -- John Mellencamp
________________________________________________________________
Bill Tantzen    University of Minnesota Libraries
612-626-9949 (U of M)    612-325-1777 (cell)